File: Resueltos_por_competidor/German_Scioville/Repetitions.java
/*
 * Autor: Scioville
 * Problema: Repetitions (https://cses.fi/problemset/task/1069)
 * Juez online: CSES
 * Veredicto: accepted  
 * hells gate arrested and shine heaven now
 * */
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Repetitions {


	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		String cadena = br.readLine();
		int a,aux;
		
		a =1;

		aux = 1;
		
		
		for(int i = 0; i < cadena.length() - 1; i++) {
		    if(cadena.charAt(i) == cadena.charAt(i+1)) {
		        a++;
		    } else {
		        aux = Math.max(aux, a);
		        a = 1;
		    }
		}
		
		aux = Math.max(aux, a);
		System.out.println(aux);
		
		
	}
}

File: Resueltos_por_competidor/Juan_Martinez/Repetitions.cpp
/*
 * Autor: Juan Martinez
 * Problema: Repetitions
 * Juez online: Cses
 * Veredicto: Accepted
 * Url: https://cses.fi/problemset/task/1069/
 */
#include <bits/stdc++.h>
using namespace std;

typedef long long int ll;

int main() {
	string s; cin>>s;
    int con = 0, maxi = 0;
    for(int i = 0; i < s.size(); i++){
        if(i == 0) con++;
        else {
            (s[i] == s[i-1]) ? con++ : con = 1;
            //cout<<con<<endl;
        }
        maxi = max(con, maxi);
    }
    cout<<maxi;
}
